From stable-bounces@linux.kernel.org  Sat Dec  9 14:56:35 2006
Date: Sun, 10 Dec 2006 09:50:36 +1100
Message-ID: <20061209225035.GA12802@gondor.apana.org.au>
From: Herbert Xu <herbert@gondor.apana.org.au>
To: Rene Herman <rene.herman@gmail.com>
Cc: torvalds@osdl.org, "David S. Miller" <davem@davemloft.net>, stable@kernel.org
Subject: dm-crypt: Select CRYPTO_CBC

As CBC is the default chaining method for cryptoloop, we should select
it from cryptoloop to ease the transition.  Spotted by Rene Herman.

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
---
 drivers/md/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.19.1.orig/drivers/md/Kconfig
+++ linux-2.6.19.1/drivers/md/Kconfig
@@ -215,6 +215,7 @@ config DM_CRYPT
 	tristate "Crypt target support"
 	depends on BLK_DEV_DM && EXPERIMENTAL
 	select CRYPTO
+	select CRYPTO_CBC
 	---help---
 	  This device-mapper target allows you to create a device that
 	  transparently encrypts the data on it. You'll need to activate
